### PR TITLE
End user security responsibilities

### PR DESCRIPTION
**Pull request by nhakmiller**
## Background

Some concerns were raised about how the permissions to modify policies/rules grant very powerful access to Panther, and how a quick glance of the product may not be indicative of what can be done with these roles. I've added a short blurb to the security & privacy section so that security conscious users can be warned and guided on how to responsibly manage their Panther deployment to not put their organization at risk.

I went ahead and tagged the whole engineering team for review to provide any additional general best practices users should follow when managing a Panther deployment, and verify the recommendations I made.

I tagged the business ops and content team to review the wording, messaging, branding, etc. of the messages.

## Changes

- Add a paragraph to the security section of the gitbook documentation

## Testing

- None

### CODE REVIEW

File: docs/gitbook/security-privacy.md [hunk 1]
```diff
@@ -6,6 +6,21 @@ Security issues identified in any of the open-source codebases maintained by Pan
 
 The data passed through Panther is always under your control and encrypted both in transit and at rest. All supporting AWS infrastructure is least-privilege and deployed with AWS CloudFormation.
 
+# The Panther Threat Model
+
+We believe that part of establishing a strong security stance is establishing your threat model. For Panther, the threat model includes attackers entirely external to the organization running Panther, as well as attackers within the organization that do not have access to Panther. That is to say, Panther is designed be secure against malicious actors attempting to abuse or sidestep the system as long as those threats do not have access to the Panther UI or admin access to the AWS account where Panther is deployed. Any attacker that does have this access has the capability to completely sidestep, break, disable, abuse, turn to evil, or anything else to Panther, and Panther does. In particular, any attacker that has the ability to edit or create arbitrary policies or rules in your Panther deployment should be considered to have full access to any and all data processed by Panther.
```
Comment (rleighton) [inline docs/gitbook/security-privacy.md:11]: Very good first sentence!

Comment (rleighton) [inline docs/gitbook/security-privacy.md:11]: s/, or anything else to Panther, and Panther does./ the Panther deployment.

Comment (rleighton) [inline docs/gitbook/security-privacy.md:11]: I would bold text: In particular, any attacker that has the ability to edit or create arbitrary policies or rules in your Panther deployment should be considered to have full access to any and all data processed by Panther.

File: docs/gitbook/security-privacy.md [hunk 2]
```diff
@@ -6,6 +6,21 @@ Security issues identified in any of the open-source codebases maintained by Pan
...
+
+# Your Responsibilities
```
Comment (rleighton) [inline docs/gitbook/security-privacy.md:13]: Very nice

File: docs/gitbook/security-privacy.md (hunk 1 above)
Comment (jacknagz) [inline docs/gitbook/security-privacy.md:11]: This is great! Can you break out the single line into multiple? The line breaks read much easier. 

Feedback:
- Remove `That is to say..`

File: docs/gitbook/security-privacy.md [hunk 3]
```diff
@@ -6,6 +6,21 @@ Security issues identified in any of the open-source codebases maintained by Pan
...
+
+Panther has been designed to be as secure as possible while still providing the core functionality of running arbitrary python rules and policies on all of your logs and cloud infrastructure. That being said, the power to write arbitrary python can easily be abused to make Panther do just about anything. It is your responsibility to ensure that the policies and rules run in your environment are trusted by you, and we recommend the following best practices to assist in this endeavor:
```
Comment (jacknagz) [inline docs/gitbook/security-privacy.md:15]: * Remove `That being said`
* Capitalize Python

**Pull request merged**